v2/yarpcprotobuf/protoc-gen-yarpc-go/main_test.go

```diff
@@ -81,13 +94,19 @@ func TestGenerate(t *testing.T) {
 					},
 				},
 			}
-			require.Equal(t, exp, act)
-
+			// Controlled by a call to `make fixtures`.
+			if os.Getenv("FIXTURES") != "" {
```

Comment:
Yea, we can definitely scrap this in the long term. I just realized that this would be helpful as we continue to make changes to our API and will inevitably need to update our fixtures.